knowledge-base/google-github-actions/auth/action-security.yml

```diff
@@ -0,0 +1,10 @@
+name: Authenticate to Google Cloud
+github-token:
+  environment-variable-name: ACTIONS_ID_TOKEN_REQUEST_TOKEN
```

Comment:
This environment variable `ACTIONS_ID_TOKEN_REQUEST_TOKEN ` is not in the examples, and so the code will not find it. I think we need to use same logic (same as token being default) as in https://github.com/step-security/secure-workflows/blob/815e14aba75030a0c717d030077b54b3755c2e1e/knowledge-base/aws-actions/configure-aws-credentials/action-security.yml#L4
We should document it somewhere. For `id-token` looks like you don't have to send the token to the Action.